crates/eframe/src/web/app_runner.rs

```diff
@@ -199,6 +203,7 @@ impl AppRunner {
 
         self.frame.info.cpu_usage = Some((now_sec() - frame_start) as f32);
 
+        let (_, repaint_after) = repaint_after[0];
```

Comment:
why only look at one element?

Reply:
This is because in a browser we cannot create multiples viewports!
So we only interested in the main viewport!

Is a way to create a new browser windows with shared state?